packages/storybook/package.json

```diff
@@ -21,6 +21,7 @@
     "@mui/x-data-grid": "5.7.0",
     "@mui/x-data-grid-generator": "5.7.0",
     "@mui/x-data-grid-pro": "5.7.0",
+    "@mui/x-data-grid-premium": "5.7.0",
```

Comment:
Maybe it's ambitious to create a new package in the same PR that adds a big feature 😁

Reply:
As explained yesterday, the PR has been approved, then rebased upon #4223 
You can find the clean diff on my repo (https://github.com/flaviendelangle/material-ui-x/pull/20) if you want to have a look at the Excel Export PR itself.

For the review concerning the Premium Plan, please review directly #4223